events/decisions_events/wc_major_zul_events.txt

```diff
@@ -0,0 +1,131 @@
+﻿namespace = zul_restoration
+
+zul_restoration.0001 = {
+	type = character_event
+	title = zul_restoration.0001.t
+	desc = {
+		desc = zul_restoration.0001.desc.intro
+		# Any Troll
+		triggered_desc = {
+			trigger = { 
+                culture = { has_cultural_pillar = heritage_zulite }  
+			}
+			desc = zul_restoration.0001.desc.troll
+		}
+		# Others
+		triggered_desc = {
+			trigger = { 
+				NOT = { culture = { has_cultural_pillar = heritage_zulite } } 
+			}
+			desc = zul_restoration.0001.desc.neutral
+		}
+	}
+	theme = realm
+
+	immediate = {
+		title:e_empire_of_zul.holder = { save_scope_as = scoped_emperor }
+		play_music_cue = "mx_cue_epic_sacral_moment"
+	}
+
+	left_portrait = {
+		character = scope:scoped_emperor
+		animation = personality_honorable
+	}
+
+	option = {
+		name = {
+			# Trolls/New Emperor
+			trigger = {
+				culture = {
+					has_cultural_pillar = heritage_zulite
+				}
+			}
+			text = zul_restoration.0001.troll
+		}
+		name = {
+			# Others
+			trigger = {
+				NOT = { 
+					culture = {
+						has_cultural_pillar = heritage_zulite
+					}
+				}
+			}
+			text = zul_restoration.0001.neutral
+		}
+		scope:scoped_emperor = {
+			set_nickname_effect = { NICKNAME = nick_the_glorious }
+		}
+	}
+
+	after = {
+		if = {
+			limit = {
+				holds_zulian_territory_trigger = yes
+				is_independent_ruler = yes
+				NOT = { root = scope:scoped_emperor }
+			}
+			trigger_event = {
+				id = zul_restoration.0002
+				days = 7
+			}
+		}
+	}
+}
+
+zul_restoration.0002 = {
+	type = character_event
+	title = zul_restoration.0001.t
+	desc = zul_restoration.0002.desc
+	theme = realm
+
+	left_portrait = {
+		character = scope:scoped_emperor
+		animation = personality_honorable
+	}
+
+	right_portrait = {
+		character = root
+		animation = thinking
+	}
+
+	immediate = {
+		title:e_empire_of_zul.holder = { save_scope_as = scoped_emperor }
+	}
+
+	# Swear Fealty
+	option = {
+		name = zul_restoration.0002.a
+		scope:scoped_emperor = {
+			send_interface_message = {
+				root = { swear_fealty_to_zul_effect = yes }
+				title = zul_swore_fealty
+				left_icon = root
+			}
+		}
+		ai_chance = {
+			base = 10
+		}
+	}
+
+	#lol, no
+	option = {
+		name = zul_restoration.0002.b
+
+		save_temporary_opinion_value_as = {
+			name = op_emp
+			target = scope:scoped_emperor 
+		}
+		ai_chance = {
+			base = 100
+			modifier = {
+				add = 20
+				scope:op_emp < 0
+			}
```

Comment:
```suggestion
		ai_chance = {
			base = 100
			modifier = {
				add = 20
				opinion = {
					target = scope:scoped_emperor 
					value < 0
				}
			}
```

Reply:
thanks, was looking for a more simple way to do that